
async function leaveServer(serverID, userID) {
    try {

    } catch (err) {
        return {type: "ERROR", msg: err}
    }
}